main.py

```diff
@@ -0,0 +1,148 @@
+import csv
+import logging
+
+
+class Account:
+    def __init__(self, name):
+        self.name = name
+        self.balance = 0
+        self.transactions = []
+
+    def add_money(self, amount):
+        self.balance += amount
+
+    def substract_money(self, amount):
+        self.balance -= amount
+
+    def add_transaction(self, date, narrative, amount, paid): # if paid is True -> person sent amount
+                                                               # if paid is False -> person received amount
+        self.transactions.append((date, narrative, amount, paid))
+        self.transactions.sort()
+
+    def print_transactions(self):
+        for transaction in self.transactions:
+            if transaction[3]:

```

Comment:
What/who is `transaction[3]`? We should either use a different data structure that is more descriptive or at least a variable with a better name 💭